core/src/main/java/com/afollestad/materialdialogs/DialogInit.java

```diff
@@ -57,7 +57,7 @@ static int getTheme(@NonNull MaterialDialog.Builder builder) {
     static int getInflateLayout(MaterialDialog.Builder builder) {
         if (builder.customView != null) {
             return R.layout.md_dialog_custom;
-        } else if (builder.items != null && builder.items.size() > 0 || builder.adapter != null) {
+        } else if (builder.items != null && builder.items.size() >= 0 || builder.adapter != null) {
```

Comment:
`builder.items.size() >= 0` isn't necessary, there's never a case where the length of an array will be *less than* 0. That check can be taken out.